docs/tutorials/connecting-to-ev3dev-with-ssh.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Connecting to Ev3dev Using SSH
```

Comment:
I know the reason but this capitalization looks strange (Ev3dev) 

Shouldn't it be ev3dev instead?